### DISCUSSION TITLE
How To Set a Value to null via REST API (Postman)?

### DISCUSSION THREAD

**Discussion by mootensai** [supabase/supabase#18037] [Questions] [answered]
Hello everyone!

I have a table with deleted_at (timestamp) and deleted_by (integer/number) columns. I have successfully set them with timestamp & user ID when deleted, but when I want to restore them by setting them to null, it doesn’t change.

I’ve tried with
deleted_at = null (empty)
deleted_by = null (empty)

or
deleted_at = null (string)
deleted_by = null (string)

or
deleted_at = NULL (string)
deleted_by = NULL (string)

but all that doesn’t change the column value. what should I do?

![image](https://github.com/supabase/supabase/assets/5844149/27b37607-e3c0-46bd-96a2-36bdd2b29ee4)

![image](https://github.com/supabase/supabase/assets/5844149/968a21d8-ff2e-484f-a350-70ba7c290403)

![image](https://github.com/supabase/supabase/assets/5844149/2e1e48d1-b2e2-4c76-b3c0-556061ac4415)

**Comment by steve-chavez** [comment]
@mootensai Please share how are you making the request.

**Reply by mootensai** [reply]
I've updated the question, please check 🙇🏻

**Comment by mootensai** [comment, marked as answer]
Oh, I can update the value with null value. Thank you!

![image](https://github.com/supabase/supabase/assets/5844149/1bb86c82-9e41-4bcf-a90e-5a81649a5a78)
